Tests/scripts/collect_tests_and_content_packs.py

```diff
@@ -1071,10 +1071,6 @@ def get_test_list_and_content_packs_to_install(files_string, branch_name, minimu
     if modified_files_with_relevant_tests:
         tests, packs_to_install = find_tests_and_content_packs_for_modified_files(modified_files_with_relevant_tests,
                                                                                   conf, id_set)
-    for pack in modified_metadata_list:
-        pack_tests = get_tests_for_pack(tools.pack_name_to_path(pack))
-        packs_to_install.add(pack)
-        tests = tests.union(pack_tests)
```

Comment:
Could you explain why this is the needed change?

Reply:
Sure. In this code, we are running over all the metadata files of the pack that was changed and add all tests of each pack to 'tests'. This is exactly what we don't want to add anymore to 'tests'. We want only to include tests of files that were changed. The tests that we do want to include are already in 'tests' 
<img width="1289" alt="Screen Shot 2020-12-07 at 18 34 08" src="https://user-images.githubusercontent.com/71635916/101377681-cda2c080-38ba-11eb-9d5c-e9cd5a5389f4.png">